src/nspawn/nspawn-stub-pid1.c

```diff
@@ -171,18 +171,12 @@ int stub_pid1(sd_id128_t uuid) {
                 /* Would love to use a switch() statement here, but SIGRTMIN is actually a function call, not a
                  * constant… */
 
-                if (si.si_signo == SIGRTMIN+3 ||
-                    si.si_signo == SIGRTMIN+4 ||
-                    si.si_signo == SIGRTMIN+13 ||
-                    si.si_signo == SIGRTMIN+14)
+                if (IN_SET(si.si_signo - SIGRTMIN, 3, 4, 13, 14))
```

Comment:
I can't say I like this, I'd prefer to keep the SIGRTMIN+n syntax. We treat that as a single word everywhere.

Reply:
Yeah. I thought it might be too aggressive. Dropped.